pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -251,21 +272,26 @@ def connection_select(context, name):
                      default='all'))
 @add_options(verify_option)
 @click.pass_obj
-def connection_add(context, **options):
+def connection_add(context, name, **options):
     """
     Add a new WBEM connection definition from specified options.
 
     Create a new WBEM connection definition in the connections file from
```

Comment:
I suggest changing this sentence to be explicit that NAME is the new name, as follows: "Create a new WBEM connection definition named NAME in the connections file...".

Reply:
DONE.

By the way, at this point I am NOT doing any paragraph reformatting to make changes easily visible in the diff.